packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -85,6 +87,13 @@ export async function fetchHelper<T>(
                 `Error ${response.status} from the server`, response.status, response);
         }
 
+        if (!parse) {
+            return {
+                headers: response.headers,
+                content: IsoBuffer.from(await response.arrayBuffer()),
```

Comment:
I do not see arrayBuffer in Response, at least the definition of typescript\lib. Are you sure it's not Node only thing?